docs/source/control_tutorials.rst

```diff
@@ -0,0 +1,349 @@
+only LED 0 will be usable. To enable the other LEDs, instead boot the machine
+using one of the pre-defined boot option dictionaries in
+:py:mod:`rig.machine_control.boot`, for example::
+
+    >>> from rig.machine_control.boot import spin3_boot_options
+    >>> mc.boot(**spin3_boot_options)
+
+To check that the system has been booted successfully, you can query it to
+retrieve its software version with
+:py:meth:`~.MachineController.get_software_version` which returns a
+:py:class:`~rig.machine_control.machine_controller.CoreInfo` named tuple::
+
+    >>> core_info = mc.get_software_version(0, 0)  # Asks chip (0, 0)
+    >>> core_info.version
+    1.33
+
+Probing for Available Resources
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The :py:meth:`~.MachineController.get_machine` method returns a
+:py:class:`~rig.machine.Machine` object describing which chips, links and cores
+are alive and also the SDRAM available::
+
+    >>> machine = mc.get_machine()
+
+This object can be directly passed to Rig's place and route utilities (e.g.
+:py:class:`rig.place_and_route.wrapper`).
+
+.. note::
+    This method simply lists *working* resources, it does not (for example)
+    exclude cores and memory which are already in use (e.g. monitor cores).
+
+Loading Applications
+^^^^^^^^^^^^^^^^^^^^
+
+The :py:meth:`~.MachineController.load_application` method unsurprisingly will
+load an application onto an arbitrary set of SpiNNaker cores. For example, the
+following code loads the specified APLX file to cores 1, 2 and 3 of chip (0, 0)
+and cores 10 and 11 of chip (0, 1)::
+
+    >>> targets = {(0, 0): set([1, 2, 3]),
+    ...            (0, 1): set([10, 11])}
+    >>> mc.load_application("/path/to/app.aplx", targets)
+
+This method alternatively accepts dictionaries mapping applications to targets,
+such as those produced by :py:class:`rig.place_and_route.wrapper`.
+
+:py:meth:`~.MachineController.load_application` verifies that all applications
+have been successfully loaded (re-attempting a small number of times if
+necessary). If not all applications could be loaded, a
+:py:exc:`~rig.machine_control.machine_controller.SpiNNakerLoadingError`
+exception is raised.
+
+Many applications require the `sync0` signal to be sent to start the
+application's event handler after loading. We can wait for all cores to reach
+the `sync0` barrier using
+:py:class:`~.MachineController.wait_for_cores_to_reach_state` and then send the
+`sync0` signal using :py:class:`~.MachineController.send_signal`::
+
+    >>> # In the example above we loaded 5 cores so we expect 5 cores to reach
+    >>> # sync0.
+    >>> mc.wait_for_cores_to_reach_state("sync0", 5)
+    5
+    >>> mc.send_signal("sync0")
+
+Similarly, after application execution, the application can be killed with::
+
+    >>> mc.send_signal("stop")
+
+Since the stop signal also cleans up allocated resources in a SpiNNaker machine
+(e.g. stray processes, routing entries and allocated SDRAM), it is desireable
```

Comment:
I think since this is an upstream bug with a fix coming very soon, I'd be tempted to leave it as just a warning in the API docs -- saves forgetting to drop it from the tutorial too...